fragment-args-ktx/src/main/kotlin/FragmentArgsDelegates.kt

```diff
@@ -757,4 +781,5 @@ private inline fun <T> Bundle.getOrElse(
     return if (containsKey(key)) getter(key) else orElse()
 }
 
-private fun noDefaultValue(): Nothing = error("No default value specified")
+private fun noDefaultValue(): Nothing =
+    error("Value for the given key is not assigned and default is not set.")
```

Comment:
I think we should add the real used key to this message. It will make easier to find bug when there is no default and key not passed too (property name is used).